ChaRo-iOS/ChaRo-iOS/Source/Views/Cells/TVC/FollowScene/FollowFollowingTVC.swift

```diff
@@ -30,19 +30,23 @@ class FollowFollowingTVC: UITableViewCell {
     }
     
     private let userNameLabel = UILabel().then {
-        $0.text = "name"
+        $0.text = ""
         $0.font = UIFont.notoSansMediumFont(ofSize: 14)
         $0.textColor = UIColor.black
     }
-    private let followButton = UIButton().then {
-        $0.setBackgroundImage(ImageLiterals.icFollowButtonGray, for: .normal)
-        $0.addTarget(self, action: #selector(followButtonClicked(_:)), for: .touchUpInside)
+    private var followButton = UIButton().then{
+        $0.setTitleColor( .mainBlue, for: .selected)
+        $0.setTitleColor( .gray30, for: .normal)
+        $0.setTitle( "팔로잉" , for: .selected)
+        $0.setTitle( "팔로워", for: .normal)
```

Comment:
```suggestion
        $0.setTitle("팔로잉" , for: .selected)
        $0.setTitle("팔로워", for: .normal)
```
엥 여기두... 왜 앞에 공백이 들어가있을까용 ㅠㅠㅠ